### PR TITLE
Adding system.json for rta-extraction

### PR DESCRIPTION
**Pull request by anbhimi**

### CODE REVIEW

File: modules/rta-extraction/service/system.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"LabsFilePath": "/Users/ananthreddy/Documents/emory_projects/json_parsing/labs/",
+	"MedsFilePath": "/Users/ananthreddy/Documents/emory_projects/json_parsing/meds/",
+	"OrdersFilePath": "/Users/ananthreddy/Documents/emory_projects/json_parsing/orders/",
+	"LabsDataClearFrequency": 30,
```
Comment (pradeeban) [inline modules/rta-extraction/service/system.json:5]: Did you mean to say "LabsDataExtractionFrequency"?

Comment (anbhimi) [reply modules/rta-extraction/service/system.json:5]: The variable is created to provide a time for clearing Labs data (in minutes). It was initialized to 30 mins for labs, meds, and order data for now.

Comment (pradeeban) [reply modules/rta-extraction/service/system.json:5]: That is too quick. You should clear the data only at 23:59 for the previous day's data, deleting data only after 24 - 48 hours later. The 30 mins frequency I mentioned was to extract the json. Different constants.

Comment (anbhimi) [reply modules/rta-extraction/service/system.json:5]: Sure will change the variable naming and will add a time constant for clearing data.

**Pull request merged**